source/features/update-pr-from-base-branch.tsx

```diff
@@ -63,20 +63,41 @@ async function addButton(mergeBar: Element): Promise<void> {
 		return;
 	}
 
-	const mergeabilityRow = $optional('.branch-action-item:has(.merging-body)');
+	const mergeabilityRow = $optional([
+		'.branch-action-item:has(.merging-body)',
+		'[aria-label="Conflicts"] [class^="MergeBoxSectionHeader-module__wrapper"]',
+	]);
+
+	const isOldView = mergeBar.classList.contains('merge-message');
+
 	if (mergeabilityRow) {
+		if (!isOldView) {
+			const conflictContent = mergeabilityRow.querySelector('h3')!;
+			if (!conflictContent.textContent.startsWith('No conflicts')) {
+				// The PR has conflicts
+				return;
+			}
```

Comment:
This shouldn't be necessary, we already make this check via API on line 62:

`prInfo.mergeable === 'CONFLICTING`